operator/src/main/resources/scripts/livenessProbe.sh

```diff
@@ -102,9 +102,11 @@ if [ $? != 0 ]; then
   exit $RETVAL
 fi
 
-copySitCfgWhileRunning /weblogic-operator/introspector ${DOMAIN_HOME}/optconfig             'Sit-Cfg-CFG--'
-copySitCfgWhileRunning /weblogic-operator/introspector ${DOMAIN_HOME}/optconfig/jms         'Sit-Cfg-JMS--'
-copySitCfgWhileRunning /weblogic-operator/introspector ${DOMAIN_HOME}/optconfig/jdbc        'Sit-Cfg-JDBC--'
-copySitCfgWhileRunning /weblogic-operator/introspector ${DOMAIN_HOME}/optconfig/diagnostics 'Sit-Cfg-WLDF--'
+if [ ! -f /weblogic-operator/introspector/domainzip.secure ]; then
```

Comment:
This looks like a fragile way to check if MII is in the picture - it relies on a relatively obscure internal behavior. Instead, how about checking the DOMAIN_SOURCE_TYPE env var? (You'll need to make sure that this env var is passed to all servers - the operator might only be passing it to the introspector pod right now...)

Reply:
Agreed to check if the env var is present and use that... If it is, unfortunately, not present then I won't want to add it as that would cause servers to roll on operator upgrade (unless we are very careful to add the env. var after the hash... sigh -- hopefully, it is already there)